todo_core/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for todo_core project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+import os.path
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+
+SECRET_KEY = os.environ.get(
+    "DJANGO_SECRET_KEY",
+    'django-insecure-$ir_+k2@1n4_$x59v231r4d0k+z%^r4^rewm27@^!86(i1mlru'
+)
+

```

Comment:
you still have this secret key visible. You should store the secret key in a separate configuration file. 
https://dev.to/themfon/how-to-protect-your-django-projects-secret-key-2ac6